### PR TITLE
Applications: nrf5340_audio: Sysbuild update for test

### PR DESCRIPTION
**Pull request by koffes**
OCT-3047
Added sysbuild config for test. Enabled error log to both UART and Shell.

### CODE REVIEW

File: tests/bluetooth/iso/Kconfig.sysbuild [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+source "${ZEPHYR_BASE}/share/sysbuild/Kconfig"
+
+config NRF_DEFAULT_IPC_RADIO
+	default y
+
+config NETCORE_IPC_RADIO_BT_HCI_IPC
```
Comment (andvib) [inline tests/bluetooth/iso/Kconfig.sysbuild:12]: I think this config should be set by `NRF_DEFAULT_IPC_RADIO`, so please try without it

File: tests/bluetooth/iso/sysbuild/ipc_radio/prj.conf [hunk 2]
```diff
@@ -0,0 +1,50 @@
+#
+# Copyright (c) 2023 Nordic Semiconductor ASA
```
Comment (andvib) [inline tests/bluetooth/iso/sysbuild/ipc_radio/prj.conf:2]:
```suggestion
# Copyright (c) 2024 Nordic Semiconductor ASA
```
nit

File: applications/nrf5340_audio/sysbuild/ipc_radio/prj.conf
Comment (andvib) [inline applications/nrf5340_audio/sysbuild/ipc_radio/prj.conf:1]: This shouldn't be changed here - update this in the tests sysbuild/ipc_radio/prj.conf instead

**Review by andvib — CHANGES_REQUESTED** [review]

**Review by rick1082 — APPROVED** [review]

**Review by alexsven — APPROVED** [review]

**Review by alexsven — APPROVED** [review]

**Review by greg-fer — CHANGES_REQUESTED** [review]
If this test is not meant for general use, as it is stated in the docs, please transform the RST file into a TXT file. 

If the test can be used by some developers, please rewrite the RST following https://docs.nordicsemi.com/bundle/ncs-latest/page/nrf/tests/crypto/README.html ( https://github.com/nrfconnect/sdk-nrf/blob/main/tests/crypto/README.rst ) as template.

File: tests/bluetooth/iso/Kconfig [hunk 3]
```diff
@@ -1,9 +1,14 @@
-config PRINT_CONN_INTERVAL
-	int "Print pr X connection interval"
-	default 100
+config PRINT_ISO_INTERVAL
+	int "Print pr X ISO interval"
```
Comment (greg-fer) [inline tests/bluetooth/iso/Kconfig:2]: What is `pr X`?

File: tests/bluetooth/iso/Kconfig [hunk 4]
```diff
@@ -1,9 +1,14 @@
...
+	default 200
 	help
 	  For each set interval, a summary will be printed and a LED toggled.
 
+config PRINT_CONN_INTERVAL
+	int "Print pr X conn interval"
```
Comment (greg-fer) [inline tests/bluetooth/iso/Kconfig:8]: What is `pr X conn`?

**Review by greg-fer — APPROVED** [review]
Pushed edits with force. Please pull changes, @koffes .

File: tests/bluetooth/iso/Kconfig (hunk 3 above)
Comment (koffes) [reply tests/bluetooth/iso/Kconfig:2]: X is the value you set. So set 5 and it prints every 5th ISO interval.

File: tests/bluetooth/iso/Kconfig (hunk 4 above)
Comment (koffes) [reply tests/bluetooth/iso/Kconfig:8]: See above.

File: tests/bluetooth/iso/Kconfig (hunk 3 above)
Comment (greg-fer) [inline tests/bluetooth/iso/Kconfig:2]:
```suggestion
	int "Print every X ISO interval"
```

File: tests/bluetooth/iso/Kconfig (hunk 4 above)
Comment (greg-fer) [inline tests/bluetooth/iso/Kconfig:8]:
```suggestion
	int "Print every X connection interval"
```

**Review by greg-fer — COMMENTED** [review]
Nits

File: tests/bluetooth/iso/modules/peripheral.c [hunk 5]
```diff
@@ -84,18 +85,17 @@ static void bt_receive_cb(struct bt_conn *conn, const uint8_t *const data, uint1
...
+		LOG_WRN("fail. added %d", (count - last_count - 1));
+	} else {
+		counts_success++;
 	}
 
 	last_count = count;
 
 	if ((count % CONFIG_PRINT_CONN_INTERVAL) == 0) {
-		LOG_INF("ACL RX. Count: %d, Failed: %d, Success: %d", count, counts_fail,
+		LOG_INF("RX: Count: %7u, Failed: %6u, Success: %7u", count, counts_fail,
```
Comment (Frodevan) [inline tests/bluetooth/iso/modules/peripheral.c:98]: Tag string as used in CI?

File: tests/bluetooth/iso/modules/iso_broadcast_sink.c [hunk 6]
```diff
@@ -366,7 +366,7 @@ static void broadcaster_sink_trd(void)
 		LOG_INF("Create BIG Sync...");
```
Comment (Frodevan) [inline tests/bluetooth/iso/modules/iso_broadcast_sink.c:366]: Tag string as used in CI?

**Review by Frodevan — CHANGES_REQUESTED** [review]

File: tests/bluetooth/iso/modules/iso_broadcast_sink.c (hunk 6 above)
Comment (koffes) [reply tests/bluetooth/iso/modules/iso_broadcast_sink.c:366]: Is this string also used by CI?

**Review by Frodevan — APPROVED** [review]

**Pull request merged**